src/App.tsx

```diff
@@ -13,8 +13,15 @@ export class App extends React.Component<{}, State> {
     movies: moviesFromServer,
   };
 
-  addMovie = (/* movie: Movie */) => {
-    // put your code here
+  addMovie = (value:Movie) => {
+    this.setState((state) => {
+      return {
+        movies: [
+          ...state.movies,
+          value,
+        ],
+      };
```

Comment:
Думаю тут можна було залишити одним рядком
```suggestion
      return { movies: [...state.movies, value] };
```